pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1372,6 +1377,7 @@ const (
 	DefTiDBMemQuotaAnalyze                         = -1
 	DefTiDBEnableAutoAnalyze                       = true
 	DefTiDBEnableAutoAnalyzePriorityQueue          = true
+	DefTiDBAnalyzeDefaultChoice                    = "ALL"
```

Comment:
```suggestion
	DefTiDBAnalyzeDefaultChoice                    = model.AllColumns.String()
```

Reply:
We cannot call a function to init the const variable. Awesome Golang!